app/views/layouts/application.html.haml

```diff
@@ -13,26 +13,28 @@
 
     = yield :head
   %body{:id => controller.controller_name, :class => controller.action_name}
-    #header
-      %div
-        = link_to t('.errbit'), root_path, :id => 'site-name'
-        = render 'shared/navigation' if current_user
-        = render 'shared/session'
-    #content-wrapper
-      #content-title{ :class => (yield :title_css_class).to_s, :style => (yield :title_style) }
-        %h1= yield :title
-        %span.meta= yield :meta
-        - if (action_bar = yield(:action_bar)).present?
-          #action-bar
-            = action_bar
-      #content
+    .row
```

Comment:
I saw some kind of problems here in application layout sidebar, the main content not fit monitor size :( I think the better solution is use bootstrap sidebar tutorials like https://ironsummitmedia.github.io/startbootstrap-simple-sidebar/ or https://ironsummitmedia.github.io/startbootstrap-sb-admin/ :)